en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/service-discovery/choreo-connect-and-service-discovery.md

```diff
@@ -0,0 +1,30 @@
+# Configure Choreo Connect with Consul Discovery (without service mesh)
+
```

Comment:
Please add an overview similar to the above covering the following.

- What are we trying to achieve with this documentation.
- Description of what Consul Discovery is.
- Why someone would want to connect Choreo Connect with Consul Discovery.

Reply:
![Screenshot from 2021-04-22 07-52-14](https://user-images.githubusercontent.com/32504465/115646453-c8ad4d80-a33f-11eb-8402-3343c7659e03.png)